Misc/NEWS.d/next/Library/2023-12-25-19-14-07.gh-issue-113471.ZQMpbI.rst

```diff
@@ -0,0 +1 @@
+Adds ability to set default content-type for no extension/unrecognized extension on http.server module.
```

Comment:
```suggestion
Adds ability to set default content-type for no extension/unrecognized extension on :mod:`http.server` module.
```

Reply:
Sure.

Reply:
Should I update my pull request with your suggestions? Not sure how, but I'll figure it out.

Reply:
Are you new to GitHub? You can just press the "Commit suggestion" button. If you have several suggestions, it is better to add them to batch and then  commit the batch -- it produces less noise ant triggers CI checks only once. Then you should pull these changes from GitHub to your machine. You can create and commit suggestions yourself, it may be convenient if you want to make only tiny changes.

Of course, you can also simply edit files on your computer, commit changes and push them. It may be more convenient if you want to amend suggestions or to make other changes.